relay-event-normalization/src/normalize/span/description/mod.rs

```diff
@@ -3,29 +3,22 @@ mod sql;
 pub use sql::parse_query;
 
 use std::borrow::Cow;
-use std::collections::BTreeMap;
 use std::path::Path;
 
 use itertools::Itertools;
-use relay_event_schema::processor::{self, ProcessingResult};
 use relay_event_schema::protocol::Span;
-use relay_protocol::{Annotated, Remark, RemarkType, Value};
 use url::Url;
 
 use crate::regexes::{
     DB_SQL_TRANSACTION_CORE_DATA_REGEX, REDIS_COMMAND_REGEX, RESOURCE_NORMALIZER_REGEX,
 };
 use crate::span::tag_extraction::HTTP_METHOD_EXTRACTOR_REGEX;
-use crate::transactions::SpanDescriptionRule;
 
 /// Attempts to replace identifiers in the span description with placeholders.
 ///
-/// The resulting scrubbed description is stored in `data.description.scrubbed`, and serves as input
-/// for the span group hash.
-pub(crate) fn scrub_span_description(span: &mut Span, rules: &Vec<SpanDescriptionRule>) {
-    let Some(description) = span.description.as_str() else {
-        return;
-    };
+/// Returns `None` if no Scrubbing can be performed.
```

Comment:
```suggestion
/// Returns `None` if no scrubbing can be performed.
```

Reply:
Whoops, forgot that I still wanted to address these comments before merging, sorry.